app/ts/McdaApp/Workspace/CurrentTab/ResultsTabs/DeterministicTab/DeterministicResults/SensitivityMeasurementsTable/SensitivityMeasurementsTableRow/SensitivityMeasurementsTableRow.tsx

```diff
@@ -17,14 +17,18 @@ export default function SensitivityMeasurementsTableRow({
 }): JSX.Element {
   const {filteredAlternatives} = useContext(CurrentSubproblemContext);
 
-  function renderCells(): JSX.Element[] {
-    return _.map(filteredAlternatives, (alternative) => (
-      <SensitivityMeasurementsTableCell
-        key={criterion.id + alternative.id}
-        criterion={criterion}
-        alternativeId={alternative.id}
-      />
-    ));
+  function Cells(): JSX.Element {
```

Comment:
I'm not sure, but I think that wrapping things in a component function like thi might introduce a bunch of extra work since it now needs to check for hooks and such, unlike a regular function. Worth investigating maybe